### PR TITLE
Add Goerli as an alias to Prater

### PR DESCRIPTION
**Pull request by terencechain**
As discussed in the bi-weekly call, we should add Goerli as an alias for Prater for better UX. This enables Goerli to be specified as Prater since the two will be merged. This allows users to configure the same network name on both consensus and execution

### CODE REVIEW

File: config/features/flags.go [hunk 1]
```diff
@@ -12,6 +12,11 @@ var (
 		Name:  "prater",
 		Usage: "Run Prysm configured for the Prater test network",
```
Comment (prestonvanloon) [inline config/features/flags.go:13]:
```suggestion
		Usage: "Run Prysm configured for the Prater test network",
		Aliases: []string{"prater"},
```

Why not do this and be done?

Comment (terencechain) [reply config/features/flags.go]: Thanks!

**Comment by terencechain** [comment]
cc @kasey

**Comment by terencechain** [comment]
tested locally and it seems to be ok

**Review by prestonvanloon — APPROVED** [review]

**Review by kasey — APPROVED** [review]

**Pull request merged**